src/diffusers/loaders.py

```diff
@@ -1499,6 +1599,52 @@ def load_lora_into_unet(cls, state_dict, network_alphas, unet, low_cpu_mem_usage
             warn_message = "You have saved the LoRA weights using the old format. To convert the old LoRA weights to the new format, you can first load them in a dictionary and then create a new dictionary like the following: `new_state_dict = {f'unet.{module_name}': params for module_name, params in old_state_dict.items()}`."
             logger.warn(warn_message)
 
+        if cls.use_peft_backend and len(state_dict.keys()) > 0:
+            from peft import LoraConfig, inject_adapter_in_model, set_peft_model_state_dict
+
+            state_dict = convert_unet_state_dict_to_peft(state_dict)
+
+            if network_alphas is not None:
+                network_alphas = convert_unet_state_dict_to_peft(network_alphas)
```

Comment:
Should this have its own method? I feel like passing `network_alphas` to a method (`convert_unet_state_dict_to_peft()`) that is supposed to accept a parameter state dict is unintuitive.

Reply:
the network alphas state dict uses the same key structure as the Unet state dict, so I preferred to simply use the same method for both

Reply:
Done!